### PR TITLE
setup cron

### LINKED ISSUE #82: cronjob inside docker

### PR DESCRIPTION
**Pull request by oliverangelil**

### CODE REVIEW

**Comment by oliverangelil** [comment]
@RicSegundo, some room for improvement in this PR:
- In the dockerfile, you'll see many instances of `/coderbyte_ishango`: maybe it should be assigned to a variable? (I also use `/coderbyte_ishango` in the crontab file). I have to specify the full path because the user is in a different directory.
- I don't really know what the difference between CMD and ENTRYPOINT is, even after reading [this](https://stackoverflow.com/a/21564990/5392289)
- probably we should be using exec form instead of shell form in the dockerfile: https://stackoverflow.com/a/42808931/5392289

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -78,6 +78,8 @@ jobs:
             --name ${{ secrets.POSTGRES_HOST }} \
             postgres
             
+            docker container rm -f ishango_app || true
+
```
Comment (RicSegundo) [inline .github/workflows/publish.yml:82]: Does this solves the problem with the exit container, and removes it if it exists?

File: dockerfile [hunk 2]
```diff
@@ -10,9 +11,12 @@ WORKDIR /coderbyte_ishango
 COPY . .
 
 # Install the required libraries and the module itself (necessary
-# to be able to run the tests with pytes)
+# to be able to run the tests with pytest)
 RUN pip install -r ./requirements.txt
 RUN pip install -e .
 
+# save environment variables to be used by cronjob (https://stackoverflow.com/a/41938139/5392289)
```
Comment (RicSegundo) [inline dockerfile:18]: This is weird 😅 so the env variables don't get saved inside the container, and the script itself cannot access them later?
Got confused here

**Review by RicSegundo — APPROVED** [review]
Great job here 🤗
I'm still super confused on the environmental variables, but I see from the stackoverflow answer you linked, that it's necessary

File: .github/workflows/publish.yml (hunk 1 above)
Comment (oliverangelil) [reply .github/workflows/publish.yml:82]: exactly. 

- if the container exists, it'll remove it
- if the container doesn't exist, it'll throw an error and then "true" will run to keep things running.

File: dockerfile (hunk 2 above)
Comment (oliverangelil) [reply dockerfile:18]: they do get saved inside the container - in `/etc/environment`. Only then are they available to the cronjob.

**Pull request merged**